frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -24,13 +37,17 @@ const ItemRow = ({item}) => {
   </div>;
 };
 
-class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalConfigPageState> {
+const GlobalConfigPage_ = connect(globalConfigPageStateToProps)(class GlobalConfigPage_ extends React.Component<Props, GlobalConfigPageState> {
```

Comment:
Merge the two `connect` calls so that there's only a single `stateToProps` function.